src/main/java/hudson/plugins/rotatews/WorkspaceBrowser.java

```diff
@@ -8,26 +8,26 @@
 import org.kohsuke.stapler.StaplerResponse;
 
 import hudson.FilePath;
-import hudson.model.AbstractBuild;
-import hudson.model.AbstractProject;
 import hudson.model.BuildBadgeAction;
 import hudson.model.DirectoryBrowserSupport;
+import hudson.model.Item;
+import hudson.model.Run;
+import jenkins.model.RunAction2;
 
-public class WorkspaceBrowser implements BuildBadgeAction {
-	AbstractBuild<?, ?> parent;
+public class WorkspaceBrowser implements BuildBadgeAction, RunAction2 {
+	transient Run<?, ?> parent;
```

Comment:
Correct, see `RunAction2` Javadoc, as of 1.485+ it is a serious problem to persist a `Run` inside an `Action`.